ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -573,11 +572,10 @@ public static void initiate_validator_exit(BeaconState state, int index) {
   public static void exit_validator(BeaconState state, int index) {
     Validator validator = state.getValidator_registry().get(index);
 
-    UnsignedLong delayed_activation_exit_epoch = get_delayed_activation_exit_epoch(get_current_epoch(state));
+    UnsignedLong delayed_activation_exit_epoch =
+        get_delayed_activation_exit_epoch(get_current_epoch(state));
     // The following updates only occur if not previous exited
-    if (validator.getExit_epoch().compareTo(delayed_activation_exit_epoch) <= 0) {
```

Comment:
This change is not in line with spec

Reply:
It's the same logic with fewer lines of code.